rubocop.yml

```diff
@@ -18,6 +18,7 @@ AllCops:
   Exclude:
   - "vendor/**/*"
   - "db/schema.rb"
+  - "spec/**/*"
```

Comment:
The only downside to this, is that it will be rendered useless when applied to Validic-Test since all files reside in the spec folder.